labs/lab-1-html-and-css/indexmobilelinked.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>chirpus inc.</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="mobile.css" />
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+<body>
+    <nav>
+        <div class="logo">
+            <span>chirpus inc.</span>
+        </div>
+        <a href="home.html">Home</a>
+        <a href="work.html">Work</a>
+        <a href="https://twitter.com/elicecil">About Us</a>
+        <a href="contact.html">Contact</a>
+        <button name="hire">Hire Us</button>
+    </nav>
+    <h1 class="sublogo">chirpus inc.</h1>
+    <section class="main">
+        <div class="intro">
+            <p class="lorem">Etiam semper euismod lectus ut egestas. Fusce vel elit in lectus fringilla bibendum. Morbi sollicitudin consequat venenatis. Morbi viverra lacinia porttitor. Nulla congue maximus dolor, et laoreet mi scelerisque vel. Donec semper sodales malesuada. Proin ultricies eros massa, ac malesuada nulla ultrices non. Mauris dolor quam, maximus ac metus ac, pellentesque dictum felis. Aliquam at mauris sodales, dictum metus sit amet, condimentum tellus. Maecenas elementum, ex vel posuere condimentum, mi elit hendrerit arcu, sit amet convallis urna nibh in urna.</p>            
+        </div>
+        <div class="circles">
+            <div class="bigboy"></div>
+            <div class="middleboy"></div>
+            <div class="smallboy"></div>
+        </div>
+    </section>
+    <footer>
```

Comment:
This is an OK place to use `<footer>`, but in general I'd reserve it for contact info/privacy policy/copyright info sort of footers.